test/fixtures/rules/std_rule_cases/L003.yml

```diff
@@ -740,7 +740,7 @@ test_pass_ignore_templated_whitespace:
     FROM my_table
   configs:
     core:
-      ignore_templated_areas: False
```

Comment:
Did the capitalized versions work? Wasn't sure if this breaking things, or just inconsistent.

Reply:
they work - it's just were getting yamllint warnings on them. I saw that when I got some actual yamllint fails - so I thought it would be better to correct them to the _recommended_ format while I was here.

Reply:
Makes sense!

YAML is a funny mixture, because it has significant whitespace like Python, but its data types are from JSON.